lib/util/merge-confidence/index.spec.ts

```diff
@@ -389,6 +404,63 @@ describe('util/merge-confidence/index', () => {
           'merge confidence API request failed - aborting run',
         );
       });
+
+      describe('parseSupportedDatasourceList()', () => {
+        let envOrg: NodeJS.ProcessEnv;
+
+        type ParseSupportedDatasourceTestCase = {
+          name: string;
+          datasourceListString: string | undefined;
+          expected: string[] | undefined;
+        };
+
+        beforeEach(() => {
+          envOrg = process.env;
+        });
+
+        afterEach(() => {
+          process.env = envOrg;
+        });
```

Comment:
simply delete the mentioned environment variable.no need to store the whole environment